com.typesafe.hocon/src/com/typesafe/config/Hocon.xtext

```diff
@@ -18,7 +18,7 @@ Root:
 
 Object:
  '{' NL*
-    ( members+=Member (',' | NL) NL* )*  
+    ( members+=Member ','? NL* )*  
```

Comment:
I am not sure this is the best fix, I think it happens to work by accident right now. Say the input is `x = 1 y = 2`, the first Member will be parsed as `x = (1 y)` and then fail on `=`. Note that `Literal` accepts `SimpleLiteral+`, so 1 or more, because Hocon lets one concatenate strings with space.

Maybe a better fix is to do the same thing as in `Root`.